src/theme/defaultTheme/input.ts

```diff
@@ -96,7 +96,7 @@ export const input = {
         suffix: "text-gray-700 ",
       },
       active: {
-        base: "bg-white-900",
```

Comment:
Why was border-gray-500 removed ?

Reply:
It was not there in Input style so added it